src/components/App/App.js

```diff
@@ -155,13 +166,97 @@ function App() {
       document.removeEventListener("mousedown", clickClose);
     };
   }, []);
+
+  const getSavedArticles = () => {
+    mainApi
+      .getSavedArticles()
+      .then((res) => {
+        setSavedCards(res);
+      })
+      .catch((err) => console.log(err));
+  };
+
+  // Delete from saved
+  const deleteArticleFromSavedNews = (articleId) => {
+    if (jwt) {
+      mainApi.removeArticle(articleId).then(() => {
+        const newSavedArticles = savedCards.filter(
+          (item) => item._id !== articleId
+        );
+        setSavedCards(newSavedArticles);
+      });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**